### DISCUSSION TITLE
Shiny Integration example unclear on how to use `daemons` when not running `shiny::runApp` directly

### DISCUSSION THREAD

**Discussion by James-G-Hill** [r-lib/mirai#177] [Q&A] [answered]
Reviewing the shiny integration document linked to below there are examples where the `daemons` function is used inside a `with` function along with `shiny::runApp` to see the daemons to run alongside the application.

However, there is no example to show how to use `daemons` when the user doesn't run `shiny::runApp` directly; for example, when using shiny server my understanding is that the server handles the `runApp` from outside of the application directory.

https://cran.r-project.org/web/packages/mirai/vignettes/shiny.html

So how to run `daemons` alongside a Shiny app that's running on shiny server or similar use case?

**Comment by shikokuchuo** [comment, marked as answer]
You would just need to include the `daemons()` call at the top level of your file, i.e. alongside the server, ui definitions.

**Reply by shikokuchuo** [reply]
I've amended the vignette so it contains an example of this in: [f6b376c](https://github.com/shikokuchuo/mirai/commit/f6b376c803cf558a32e0a731fb6acec6482f9f9d).

**Comment by James-G-Hill** [comment]
Thanks!
